### DISCUSSION TITLE
android framework java代码该如何实现C代码compile_commands.json精确定位代码的效果

### DISCUSSION THREAD

**Discussion by djfyz** [neoclide/coc.nvim#4552] [General]
大佬好，我是从事Android手机开发相关工作的，工作内容设计底层驱动C/C++和android framework，主要是 java。
目前用coc.nvim插件的compile_commands.json文件可以很精确的识别C的定义和调用，但我对java不是很熟悉，想问下针对Android framework这种大型java代码工程，有没有一种类似compile_commands.json的方法，可以把所有相关源文件都可以精确的识别到，谢谢！

**Comment by fannheyward** [comment]
你可以试试 https://github.com/neoclide/coc-java ，但是不确定 jdt.ls 对 Android framework 这种大型项目是否能正常工作。
